cell.go

```diff
@@ -114,21 +114,21 @@ func (c *Cell) SetFloatWithFormat(n float64, format string) {
 
 var timeLocationUTC, _ = time.LoadLocation("UTC")
 
-func timeToUTCTime(t time.Time) time.Time {
+func TimeToUTCTime(t time.Time) time.Time {
```

Comment:
I'm not sure why you needed to change this, the only consumer is in the same module below.